migrations/m161203_155345_add_university_name_column_to_university_table.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use yii\db\Migration;
+
+/**
+ * Handles adding university_name to table `university`.
+ */
+class m161203_155345_add_university_name_column_to_university_table extends Migration
+{
+    /**
+     * @inheritdoc
+     */
+    public function up()
+    {
+        $this->addColumn('university', 'university_name', $this->string());
```

Comment:
You don't need to use "university_" prefix for all the fields. It's enought to call them "id", "name" (if they are inside of the "university" table. But in other tables you need to specify the foreign table while naming the id fields (for example "university_id" inside the "student" table)